src/benchmark/file_io_read_mmap_benchmark.cpp

```diff
@@ -48,14 +65,17 @@ void FileIOMicroReadBenchmarkFixture::mmap_read_single_threaded(benchmark::State
     state.ResumeTiming();
 
     // Remove memory mapping after job is done.
-    Assert((munmap(map, NUMBER_OF_BYTES) == 0), fail_and_close_file(fd, "Unmapping failed: ", errno));
+    if (mapping_type == MMAP) {
+      Assert((munmap(map, NUMBER_OF_BYTES) == 0), fail_and_close_file(fd, "Unmapping failed: ", errno));
+    } else {
```

Comment:
Could you work here with else if? See line 40-46 
I think it reduces reading complexity when you know, what the else case means
What do you think?